groups/rds/main.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = "==0.14"
```

Comment:
We don't have a 0.14 version of the Terraform Runner, so you should look to reduce the required version to be at least 0.13.

e.g: `">=0.13"`